File: app/src/main/java/com/robotnec/budget/fragments/AboutFragment.java
package com.robotnec.budget.fragments;

import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v4.app.Fragment;
import android.support.v7.widget.Toolbar;
import android.view.View;

import butterknife.Bind;
import com.robotnec.budget.R;
import com.robotnec.budget.core.mvp.presenter.AboutPresenter;
import com.robotnec.budget.core.mvp.view.AboutView;

/**
 * @author zak <zak@swingpulse.com>
 */
public class AboutFragment extends BasePresenterFragment<AboutPresenter> implements AboutView {

    @Bind(R.id.toolbar)
    Toolbar toolbar;

    public static Fragment newInstance() {
        return new AboutFragment();
    }

    @Override
    protected int getLayoutId() {
        return R.layout.fragment_about;
    }

    @Override
    protected AboutPresenter createPresenter() {
        return new AboutPresenter(this);
    }

    @Override
    public void onViewCreated(View view, @Nullable Bundle savedInstanceState) {
        super.onViewCreated(view, savedInstanceState);
        initToolbarToggle(toolbar);
    }
}

File: app/src/main/java/com/robotnec/budget/app/fragments/AboutFragment.kt
package com.robotnec.budget.app.fragments

import androidx.fragment.app.Fragment
import com.robotnec.budget.R
import com.robotnec.budget.core.mvp.presenter.AboutPresenter
import com.robotnec.budget.core.mvp.view.AboutView

/**
 * @author zak zak@swingpulse.com>
 */
class AboutFragment : BasePresenterFragment<AboutPresenter>(), AboutView {

    companion object {
        fun newInstance(): Fragment {
            return AboutFragment()
        }
    }

    override val layoutId: Int
        get() = R.layout.fragment_about

    override fun createPresenter(): AboutPresenter {
        return AboutPresenter(this)
    }
}
